src/L2TokenGateway.sol

```diff
@@ -163,14 +163,14 @@ contract L2TokenGateway is ITokenGateway, ICustomGateway, L2ArbitrumMessenger {
         uint256 amount,
         bytes memory data
     ) public pure returns (bytes memory outboundCalldata) {
-        outboundCalldata = abi.encodeWithSelector(
-            ITokenGateway.finalizeInboundTransfer.selector,
+        outboundCalldata = abi.encodeCall(
+            ITokenGateway.finalizeInboundTransfer, (
             token,
             from,
             to,
             amount,
             abi.encode(0, data) // using 0 for exitNum as exit redirection is not supported
-        );
+        ));
```

Comment:
I'd suggest doing the named params for sendTxToL1 as we did in the other side Gateway.

Reply:
I mean I'd say we do on both or none.

Reply:
Yes, done.